antora-ui-camel/src/css/catalog.css

```
@@ -0,0 +1,26 @@
+.catalog-container {
```

We prefix all content categories with a class to minimize the chance of rule collision and increase the specificity of the rule. With that it would be better to use `.catalog .container` selector.

```suggestion
.catalog .container {
```